### PR TITLE
[New Specs] ls_var_lib_pcp

### PR DESCRIPTION
**Pull request by mohitkumarrh**
Signed-off-by: Mohit Kumar <mohikuma@REDHAT.COM>

### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### Complete Description of Additions/Changes:

*Add your description here*

### CODE REVIEW

File: insights/specs/__init__.py [hunk 1]
```diff
@@ -339,6 +339,7 @@ class Specs(SpecSet):
     ls_usr_sbin = RegistryPoint(filterable=True)
     ls_var_cache_pulp = RegistryPoint()
     ls_var_lib_mongodb = RegistryPoint()
+    ls_var_lib_pcp = RegistryPoint()
```
Comment (xiangce) [inline insights/specs/__init__.py:342]: Please adjust its location per the alphabet order.

File: insights/specs/default.py [hunk 2]
```diff
@@ -328,6 +328,7 @@ class DefaultSpecs(Specs):
     ls_usr_lib64 = simple_command("/bin/ls -lan /usr/lib64")
     ls_var_cache_pulp = simple_command("/bin/ls -lan /var/cache/pulp")
     ls_var_lib_mongodb = simple_command("/bin/ls -la /var/lib/mongodb")
+    ls_var_lib_pcp = simple_command("/bin/ls -la /var/lib/pcp")
```
Comment (xiangce) [inline insights/specs/default.py:331]: The same comment as above.

File: insights/specs/insights_archive.py [hunk 3]
```diff
@@ -134,6 +134,7 @@ class InsightsArchiveSpecs(Specs):
     ls_usr_lib64 = simple_file("insights_commands/ls_-lan_.usr.lib64")
     ls_var_cache_pulp = simple_file("insights_commands/ls_-lan_.var.cache.pulp")
     ls_var_lib_mongodb = simple_file("insights_commands/ls_-la_.var.lib.mongodb")
+    ls_var_lib_mongodb = simple_file("insights_commands/ls_-la_.var.lib.pcp")
```
Comment (xiangce) [inline insights/specs/insights_archive.py:137]: `ls_var_lib_pcp`

File: insights/tests/parsers/test_ls_var_lib_pcp.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+def test_ls_var_lib_pcp():
+    ls_var_lib_pcp = LsVarLibPcp(context_wrap(LS_VAR_LIB_PCP, path="insights_commands/ls_-la_.var.lib.pcp"))
+    assert ls_var_lib_pcp.dirs_of('/var/lib/pcp') == ['.', '..', 'pmcd', 'pmie', 'pmdas']
+    journal = ls_var_lib_pcp.dir_entry('/var/lib/pcp', 'pmdas')
+    assert journal is not None
+
+
+def test_ls_var_lib_pcp_doc_examples():
+    env = {
+        'LsVarLibPcp': LsVarLibPcp,
```
Comment (xiangce) [inline insights/tests/parsers/test_ls_var_lib_pcp.py:26]: This key-value pair is not required for doc test.

File: insights/parsers/ls_var_lib_pcp.py [hunk 5]
```diff
@@ -0,0 +1,39 @@
+"""
+LsVarLibPcp - command ``ls -la /var/lib/pcp``
+=====================================================
```
Comment (xiangce) [inline insights/parsers/ls_var_lib_pcp.py:3]: Please make the `===` line the same length as the title.

**Review by xiangce — CHANGES_REQUESTED** [review]

**Comment by mohitkumarrh** [comment]
@xiangce - Thanks for the review, I'll make all the changes.

File: insights/parsers/ls_var_lib_pcp.py [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+    True
+    >>> ls_var_lib_pcp.dir_entry('/var/lib/pcp', 'pmdas')['type']
+    'd'
+"""
+
+
+from insights.specs import Specs
+
+from .. import CommandParser, parser
+from .. import FileListing
```
Comment (psachin) [inline insights/parsers/ls_var_lib_pcp.py:33]: @mohitkumarrh 

Please avoid relative imports, instead use,
```python
from insights import CommandParser, parser
from insights.core import FileListing
```

**Comment by mohitkumarrh** [comment]
py.test:
```
➜  insights-core git:(new_specs_ls_var_lib_pcp) ✗ py.test -svk ls_var_lib_pcp --cov=insights.parsers.ls_var_lib_pcp
=========================================================================== test session starts ===========================================================================
platform darwin -- Python 3.9.1, pytest-7.1.2, pluggy-1.0.0 -- /Users/mohitkumar/.pyenv/versions/3.9.1/bin/python3.9
cachedir: .pytest_cache
rootdir: /Users/mohitkumar/Insights/insights-core, configfile: setup.cfg, testpaths: insights
plugins: cov-3.0.0
collecting 603 items                                                                                                                                                      
Downloading a fresh and hot uploader.json...
collected 2942 items / 2940 deselected / 2 selected                                                                                                                       

insights/tests/parsers/test_ls_var_lib_pcp.py::test_ls_var_lib_pcp PASSED
insights/tests/parsers/test_ls_var_lib_pcp.py::test_ls_var_lib_pcp_doc_examples PASSED

---------- coverage: platform darwin, python 3.9.1-final-0 -----------
Name                                 Stmts   Miss Branch BrPart  Cover   Missing
--------------------------------------------------------------------------------
insights/parsers/ls_var_lib_pcp.py       6      0      2      0   100%
--------------------------------------------------------------------------------
TOTAL                                    6      0      2      0   100%


=================================================================== 2 passed, 2940 deselected in 6.57s ====================================================================

```

**Comment by mohitkumarrh** [comment]
@xiangce @psachin - I have updated the PR as you suggested, please review again

**Comment by psachin** [comment]
@mohitkumarrh This looks good to me now.

**Review by psachin — APPROVED** [review]

**Comment by psachin** [comment]
/ Pending specs approval /

File: insights/parsers/ls_var_lib_pcp.py [hunk 7]
```diff
@@ -0,0 +1,38 @@
+"""
+LsVarLibPcp - command ``ls -la /var/lib/pcp``
+=============================================
+
+The ``ls -la /var/lib/pcp`` command provides information for the listing of the ``/var/lib/pcp`` directory.
+
+Attributes:
+    file_permissions (list): list of `FilePermissions` objects for every file from the output
```
Comment (xiangce) [inline insights/parsers/ls_var_lib_pcp.py:8]: I don't think this parser provides this attribute.

Comment (mohitkumarrh) [reply insights/parsers/ls_var_lib_pcp.py]: @xiangce I have corrected this mistake, please have a look again

**Review by xiangce — APPROVED** [review]

**Pull request merged**